.github/workflows/ci.yml

```diff
@@ -30,6 +30,13 @@ jobs:
         with:
           tool: cargo-hack,taplo-cli
 
+      - name: Check C formatting
+        run: |
+          find ./ -type f \
+            -not \( -path "./xtask/libbpf/*" -o -path "./target/*" \) \
```

Comment:
git ls-files -- '*.c' '*.h'

Then you don't need all the filtering

Reply:
Thanks